### DISCUSSION TITLE
Vanilla respawn shuttle buff

### DISCUSSION THREAD

**Discussion by Cogs16** [FakeFishGames/Barotrauma#5677] [Ideas and suggestions]
In casual gamemodes respawn shuttles are often destroyed 50% of the time before reaching the main sub. Most of them are fragile, slow, and can be easily sunk by opening the airlock. When 16 people are on a crowded slow sub, it becomes very easy to interfere with its operations. In order to fix this respawn shuttles should be buffed, with faster engines, pumps, and strong hulls. I think in future when more advanced sub nav AI is implemented seeing automatic respawn shuttles would also help with griefers crashing them into walls.

**Comment by Cogs16** [comment]
Weak shuttles can also lead to the main submarine being sunk, because it crashes on top of them and then pushes the main sub beyond crush depth.

**Comment by thegrb93** [comment]
I think it being weak is good. It tests the respawning players' mettle in whether they deserve to reach the main sub again or not. Admins can and should take care of the trouble makers.

There's no drama if the shuttle is impervious to failure.

**Reply by Cogs16** [reply]
Here and there is fine, however if people get caught in an endless loop because of only 1 bad apple the game is no longer fun. Also, there is nothing that can be done if a swarm of creatures spawns nearby, and because there is no access control on the navconsole its too easy to grief. The base game should not have to rely on admin intervention, additionally imo the drama should be around the main sub, where the main parts of gameplay happen, not on the vehicle for respawning.

**Reply by thegrb93** [reply]
> The base game should not have to rely on admin intervention

Depends on how the server wishes its play style to be. Either way, what happens won't matter much because we can just use whatever favorite shuttle we want from workshop.
